beacon-chain/rpc/validator/proposer_test.go

```diff
@@ -2105,27 +2105,6 @@ func TestProposer_DeleteAttsInPool_Aggregated(t *testing.T) {
 	assert.Equal(t, 0, len(atts), "Did not delete unaggregated attestation")
 }
 
-func TestProposer_SortProfitableAtts(t *testing.T) {
```

Comment:
Any reason this test is removed?

Reply:
It is not removed, it is *moved* (`proposer_utils_test.go` file was added, and this test, correctly, was moved there).

Reply:
Here is test's new location: https://github.com/prysmaticlabs/prysm/pull/8063/files#diff-8b5612101d61e04a5d66fe163e80b50752d2c1ab84e319a65a2405571543c02fR14